ArcPicker/ArcPicker.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Agate styled arc picker components and behaviors.
+ *
+ * @example
+ * <ArcPicker backgroundColor="#444444" endAngle={200} foregroundColor="#eeeeee" selectionType="single" startAngle={0} />
```

Comment:
Should we define `backgroundColor` and `foregroundColor` here?

Reply:
It was a requirement that the user could choose the background and foreground colors. https://github.com/enactjs/agate/pull/340#discussion_r494051437
Therefore these props need to be put in the example.

Reply:
I meant, the default values are there, so no need to add.
If you intended to add custom bgColor and fgColor, it's fine.